patroni/ha.py

```diff
@@ -846,17 +987,24 @@ def check_failsafe_topology(self) -> bool:
         return all(results)
 
     def is_lagging(self, wal_position: int) -> bool:
-        """Returns if instance with an wal should consider itself unhealthy to be promoted due to replication lag.
+        """Checks if node should consider itself unhealthy to be promoted due to replication lag.
 
         :param wal_position: Current wal position.
-        :returns True when node is lagging
+        :returns `True` when node is lagging
         """
         lag = (self.cluster.last_lsn or 0) - wal_position
         return lag > self.global_config.maximum_lag_on_failover
 
     def _is_healthiest_node(self, members: Collection[Member], check_replication_lag: bool = True) -> bool:
-        """This method tries to determine whether I am healthy enough to became a new leader candidate or not."""
-
+        """This method tries to determine whether the current node is healthy enough to became a new leader candidate.
+
+        :param members: the list of nodes to check against
+        :param check_replication_lag: whether to take the replication lag into account.
+                                      If the lag exceeds configured threshold the node disqualifies itself.
+        :returns: `True` in case if the node is eligible to become the new leader. Since this method is executed
+                  on multiple nodes independently it could happen that many nodes will count themselves as
+                  healthiest because they received/replayed up to the same LSN, but it is totally fine.
```

Comment:
```suggestion
        :returns: `True` if the node is eligible to become the new leader. Since this method is executed
                  on multiple nodes independently it is possible that multiple nodes could count 
                  themselves as the healthiest because they received/replayed up to the same LSN,
                  but this is totally fine.
```